Dockerfile

```diff
@@ -22,13 +21,18 @@ RUN apk --update add \
     rm /var/cache/apk/* && \
     ln -s /usr/bin/php7 /usr/bin/php
 
+COPY composer.json /usr/src/app/composer.json
+COPY composer.lock /usr/src/app/composer.lock
+
 RUN apk --update add curl && \
     curl -sS https://getcomposer.org/installer | php && \
     ./composer.phar install && \
     apk del curl && \
     rm /usr/src/app/composer.phar \
        /var/cache/apk/*
 
+COPY bin/build-content /usr/src/app/bin/build-content
```

Comment:
What do you think of adding the `bin/` directory to the COPY directive above to avoid this extra layer? It's rare that these bin files change so I think we can take the cache miss  when they do.